hyperwallet/tests/test_encryption.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+
+import unittest
+import time
+import json
+import os.path
+
+from jwcrypto import jwk, jws as cryptoJWS
+from jwcrypto.common import json_encode
+from hyperwallet.exceptions import HyperwalletException
+from hyperwallet.utils.encryption import Encryption
+from django.core.validators import URLValidator
+from django.core.exceptions import ValidationError
```

Comment:
Could you use URLValidator to avoid adding django dependency 
https://github.com/adonisnafeh/urlvalidator

Reply:
Removed Django dependency